guidelines/README.md

```diff
@@ -0,0 +1,5 @@
+# Guidelines
+
+The objective of guidelines is to share knowledge between club leaders in a more
```

Comment:
It differentiates what to put in workshops and what to put in guidelines. Is that unclear from the way it is worded?